tsc/meetings/2019-08-29.md

```diff
@@ -0,0 +1,130 @@
+Minutes from 26th OpenVDB TSC meeting, July 25, 2019
+
+Attendees: *Nick* A., *Dan* B., *Jeff* L., *Ken* M., *Peter* C.
+
+Additional Attendees: Sean McDuffee
+
+Agenda:
+
+1) Quorum
+2) Secretary
+3) Round Robin
+4) Release Schedule for 6.2.0
+5) levelSetVolume
+6) VDB Smooth doesn't densify
+7) Bitscan/popcount
+8) Deprecation
+9) Combine SOP Performance
+10) Tree Refactoring
+11) Delayed Loading
+12) Next Meeting
+
+1) Quorum was confirmed.
+
+2) Secretary was Jeff Lait
+
+3) Round Robin
+
+A quick round robin to identify what is being worked on and what may
+make it for 6.2.
+
+Ken: Working on velocity extension.  Read-Only grid may end up being internal
+only.  Has found interesting results by making a write-only grid that allows
+better multithreaded scaling than usual multiple grids + merge.
+
+Nick: AX integration.  Feature branch is hoped to exist in a week or so.
+As well, CMake as required.  Should anistropic rasterization be higher
+priority?  In response to a later question, MPI will not be available with
+current priorities.
+
+Dan: Point partitioning improvements.  C++11 changes.  30-40% better performance
+on some datasets.  Merging of grids is open questions.  Likewise, refactoring
+tree hierarchy and improving combine SOP.
+
+Jeff: Hopes to get VDB Activate moved to OpenVDB.  6.1 integration into
+Houdini 18.0.
+
+Peter: Principle axis extraction from volumes, moment of inertia.  Sharpen
+filter.  Discussion on sharpen filter.  Does support non-level set, consensus
+was it should be kept regardless of differences with Ken's version.
+
+4) Release Schedule for 6.2.0
+
+No one had any critical items to be done for 6.2.0.
+
+5) levelSetVolume
+
+VDB Clip in 6.X leaves different values in inactive voxels, causing volume
+computation to change.  Consensus is that such half-open SDF don't have
+a proper volume so this is acceptable.
+
+Peter notes that there is an error computing volume of entirely empty
```

Comment:
Not exactly: the issue is that empty bounding boxes are represented by _max_ < _min_ (because `CoordBBox` bounds are closed intervals rather than half-open intervals), but the bounding box volume computation doesn't check for that case.

Reply:
Hopefully I have this clarified.

Reply:
PR #512 includes the fix.